### PR TITLE
Add new Dockerfile

### PR DESCRIPTION
**Pull request by Id3aFly**
This adds a Dockerfile. As I am no Docker professional this should be reviewed. However, I tried to apply my knowledge for best security within.

### CODE REVIEW

**Comment by tschmidtb51** [comment]
Thank you for your contribution. Could you please clarify what the purpose of the docker file would be? Running in production? (That shouldn't be done with `npm run dev`.) Please also add a sentence or two about Docker in the Readme. Thanks!

**Comment by Id3aFly** [comment]
Yes, the purpose is to run this in production. As the [readme](https://github.com/secvisogram/csaf-validator-service#run-server )tells to use `npm run dev` I was using it as well. Is there a better way? 

I will update the Readme.

**Comment by tschmidtb51** [comment]
Okay. I see where you come from - but that is just for development purposes... Please review the [Getting started section](https://github.com/secvisogram/csaf-validator-service#getting-started) for the production procedure.

**Comment by mfd2007** [comment]
I improved the docker file to use the instructions for for production use.

An additional config file would be needed:
/config/local-production.json
```
{
  "ip": "0.0.0.0",
  "port": 8082
}
```

**Comment by mfd2007** [comment]
I would suggest adding at least:

## Docker

Build docker image 

```sh
docker build -t csaf/validator-service:1.0.0 .
```

Start container
```sh
docker run -d -p 8082:8082 --name csaf-validator-service csaf/validator-service:1.0.0
```

**Comment by Id3aFly** [comment]
> I would suggest adding at least:
> 
> ## Docker
> Build docker image
> 
> ```shell
> docker build -t csaf/validator-service:1.0.0 .
> ```
> 
> Start container
> 
> ```shell
> docker run -d -p 8082:8082 --name csaf-validator-service csaf/validator-service:1.0.0
> ```

I added your suggestion.

**Comment by Id3aFly** [comment]
I'm receiving following error:
image

**Comment by Id3aFly** [comment]
> I'm receiving following error: image
Fixed with latest commit. Semicolon was missing.

**Review by mfd2007 — DISMISSED** [review]
Seems good to me.

**Comment by tschmidtb51** [comment]
> ```shell
> docker build -t csaf/validator-service:1.0.0 .
> ```

Why exactly do we statically reference `1.0.0`?

**Comment by tschmidtb51** [comment]
> ```shell
> docker build -t csaf/validator-service:1.0.0 .
> ```

Why exactly do we statically reference `1.0.0`?

**Comment by tschmidtb51** [comment]
@pdamian: Any additional comments on the security from your side?

File: README.md [hunk 1]
```diff
@@ -95,6 +95,20 @@ npm test
...
+```sh
+docker build -t csaf/validator-service:1.0.0 .
+```
+
+Start container
+
+```sh
+docker run -d -p 8082:8082 --name csaf-validator-service csaf/validator-service:1.0.0
+```
+
```
Comment (tschmidtb51) [inline README.md:111]: This is a file to use it in production right? Maybe, we should add that as a subsection under "Getting started" and refer to this in the sentence `To manage the process you can use Docker or an init system of your choice. An example Docker setup is provided below.`

@mfd2007, @Id3aFly : Thoughts?

Comment (mfd2007) [reply README.md:111]: We can 

> > ```shell
> > docker build -t csaf/validator-service:1.0.0 .
> > ```
> 
> Why exactly do we statically reference `1.0.0`?

This was just an example. We can it to `x.y.z` to make it more general. I don't want to use `latest` as tag.

Comment (mfd2007) [reply README.md:111]: > This is a file to use it in production right? Maybe, we should add that as a subsection under "Getting started" and refer to this in the sentence `To manage the process you can use Docker or an init system of your choice. An example Docker setup is provided below.`
> 
> @mfd2007, @Id3aFly : Thoughts?

Would be okay for me.

Comment (tschmidtb51) [reply README.md:111]: > We can
> 
> > > ```shell
> > > docker build -t csaf/validator-service:1.0.0 .
> > > ```
> > 
> > 
> > Why exactly do we statically reference `1.0.0`?
> 
> This was just an example. We can it to `x.y.z` to make it more general. I don't want to use `latest` as tag.

Is there a special reason why you don't want to use the `latest` tag?

Comment (mfd2007) [reply README.md:111]: `latest` is ambiguous. Let's just skip the tag and write `docker build -t csaf/validator-service .` in the readme.

Comment (Id3aFly) [reply README.md:111]: I don't have any preference.

File: README.md [hunk 2]
```diff
@@ -95,6 +95,20 @@ npm test
 
 [(back to top)](#bsi-secvisogram-csaf-validator-service)
 
+## Docker
+
+Build docker image
+
+```sh
+docker build -t csaf/validator-service:1.0.0 .
```
Comment (tschmidtb51) [inline README.md:103]: As indicated by @mfd2007:
```suggestion
docker build -t csaf/validator-service .

File: README.md [hunk 3]
```diff
@@ -95,6 +95,20 @@ npm test
...
+```
+
+Start container
+
+```sh
+docker run -d -p 8082:8082 --name csaf-validator-service csaf/validator-service:1.0.0
```
Comment (tschmidtb51) [inline README.md:109]: I guess this one has to be changed as well...
```suggestion
docker run -d -p 8082:8082 --name csaf-validator-service csaf/validator-service

**Review by tschmidtb51 — COMMENTED** [review]
@mfd2007: Are those changes correct?

**Review by mfd2007 — APPROVED** [review]

**Review by tschmidtb51 — APPROVED** [review]
LGTM

**Pull request merged**